pom.xml

```diff
@@ -62,11 +62,11 @@
         </pluginRepository>
     </pluginRepositories>
     <properties>
-        <jenkins.version>2.62</jenkins.version>
+        <jenkins.version>2.73.3</jenkins.version>
```

Comment:
Unrelated, but easier to remember LTS baselines.

Reply:
Let's not bump the baseline unless there's a need to - this limits who can take advantage of new plugin releases.

Reply:
Bumped only to the nearest LTS line. There is no reason for anyone to still be using intermediate weeklies. If they are, they need to update.